### PR TITLE
Update config and babel for ie11

### PR DESCRIPTION
**Pull request by skjnldsv**
Signed-off-by: John Molakvoæ (skjnldsv) <skjnldsv@protonmail.com>

### CODE REVIEW

File: settings/.babelrc [hunk 1]
```diff
@@ -1,15 +0,0 @@
...
-	"presets": [
-		[
-			"env",
-			{
-				"targets": {
-					"browsers": ["last 2 versions", "ie >= 11"]
-				},
-				"modules": false,
-        "blacklist": ["useStrict"],
-        "useBuiltIns": true
```
Comment (nickvergessen) [inline settings/.babelrc]: those two not anymore?

**Comment by nickvergessen** [comment]
Same should happen to all the apps in apps/ too I guess?

File: settings/.babelrc (hunk 1 above)
Comment (skjnldsv) [reply settings/.babelrc]: No, they were not needed :)

**Comment by skjnldsv** [comment]
>Same should happen to all the apps in apps/ too I guess?

@nickvergessen Yes :)

**Review by nickvergessen — APPROVED** [review]

**Comment by nickvergessen** [comment]
@skjnldsv 14.0.1 backport? or 14.0.2?

**Comment by rullzer** [comment]
conflicts

**Review by rullzer — APPROVED** [review]
Sure fine by me

**Comment by MorrisJobke** [comment]
@skjnldsv Mind to do the backport to stable14?

**Comment by skjnldsv** [comment]
Backport in https://github.com/nextcloud/server/pull/11490

**Pull request merged**